homeassistant/components/rainforest_raven/coordinator.py

```diff
@@ -23,6 +23,7 @@
 type RAVEnConfigEntry = ConfigEntry[RAVEnDataCoordinator]
 
 _LOGGER = logging.getLogger(__name__)
+_DEVICE_TIMEOUT = 5
```

Comment:
Making this a constants means we can override it in the tests, and reduce the test from 5s to 0.1s